docs/developers/terms/ratio.md

```diff
@@ -1,19 +1,26 @@
 ---
 id: ratio
 title: Ratio
-hoverText: The ratio between the amount promised by an offer and the amount it requests.
+hoverText: The ratio $wants/gives$ between the amount $gives$ promised by an offer and the amount $wants$ it requests.
 ---
 
-The _ratio_ corresponds to the comparison between the amount promised by an offer and the amount it requests.
+The _ratio_ $wants/gives$ between the amount $gives$ promised by an offer and the amount $wants$ it requests. The unit of ratios on an %%offer list|offer-list%% `outbound_tkn-inbound_tkn` is thus `inbound_tkn/outbound_tkn`.
+
+A ratio corresponds to a %%price|price%% in the follow way:
+
+* For an _ask_, $price = ratio$
+* For a _bid_, $price = 1/ratio$.
```

Comment:
Since we mention "inbound_tkn" we are nearing the terminology of solidity contracts, so should we note here that price is typically in display units while ratio for solidity is raw?

Reply:
Addressed in #203